<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="addRowButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>新增一行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeRowButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>510</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除选中行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addColumnButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>440</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>新增一列</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeColumnButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>510</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除选中列</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newColumnHeaderLineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>480</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入新增列表名</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>481</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
